src/main/kotlin/me/alexflipnote/kawaiibot/commands/Slap.kt

```diff
@@ -19,7 +20,7 @@ class Slap : ICommand {
             m.user.idLong == ctx.jda.selfUser.idLong ->
                 ctx.send("**${ctx.author.name}** we can no longer be friends ;-;")
             m.user.idLong == ctx.author.idLong ->
-                ctx.send("https://media0.giphy.com/media/1M9fmo1WAFVK0/giphy.gif") // TODO: bundle this maybe
+                ctx.channel.sendFile(Helpers.getImageStream("images/butwhy.webp"), "butwhy.webp").queue()
             else -> {
```

Comment:
huh.. webp?
I know what it is but, I don't think Apple users (specially iPhone/iPad) supports this format